Bow OpenAPI/Generator/SwaggerClientGenerator.swift

```diff
@@ -31,11 +31,13 @@ class SwaggerClientGenerator: ClientGenerator {
         return EnvIO { _ in runSwagger() }
     }
     
-    private func reorganizeFiles(in output: String) -> EnvIO<FileSystem, APIClientError, ()> {
+    private func reorganizeFiles(in outputPath: GeneratorOutput, fromTemplate templatePath: String) -> EnvIO<FileSystem, APIClientError, ()> {
         EnvIO { fileSystem in
             binding(
-                |<-fileSystem.move(from: "\(output)/SwaggerClient/Classes/Swaggers", to: output),
-                |<-fileSystem.remove(from: output, files: "Cartfile", "AlamofireImplementations.swift", "Models.swift", "git_push.sh", "SwaggerClient.podspec", "SwaggerClient", ".swagger-codegen", ".swagger-codegen-ignore", "JSONEncodableEncoding.swift", "JSONEncodingHelper.swift"),
+                |<-fileSystem.move(from: "\(outputPath.sources)/SwaggerClient/Classes/Swaggers", to: outputPath.sources),
+                |<-fileSystem.remove(from: outputPath.sources, files: "Cartfile", "AlamofireImplementations.swift", "Models.swift", "git_push.sh", "SwaggerClient.podspec", "SwaggerClient", ".swagger-codegen", ".swagger-codegen-ignore", "JSONEncodableEncoding.swift", "JSONEncodingHelper.swift"),
+                |<-fileSystem.rename("APIConfiguration", itemAt: "\(outputPath.sources)/APIHelper.swift"),
```

Comment:
According to my comment above, this file name should include the `.swift` extension.